helpers/middlewares.js

```diff
@@ -0,0 +1,8 @@
+exports.exceptionMiddleware = async function exceptionMiddleware(err, req, res, next) {
+    res.status(500);
```

Comment:
Also add 'scrapy-puppeteer-service-context-id' header. This makes the context reachable if there are multiple service instances behind haproxy.